corehq/apps/reports/models.py

```diff
@@ -235,17 +235,10 @@ def create_jwt(self):
         return token
 
 
-class TableauGroup(models.Model):
-    server = models.ForeignKey(TableauServer, on_delete=models.CASCADE)
-    name = models.CharField(max_length=64)
-    group_id = models.CharField(max_length=32)
-
-
 class TableauUser(models.Model):
     server = models.ForeignKey(TableauServer, on_delete=models.CASCADE)
-    username = models.CharField(max_length=255)
+    username = models.CharField(max_length=255, unique=True)
```

Comment:
would it be possible to have the same username associated with different 'server'?